package-lock.json

```diff
@@ -1,13 +1,1865 @@
 {
   "name": "redis-fast-driver",
-  "version": "2.0.2",
+  "version": "2.1.0",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
Package-lock should be generated only for prod dependencies, please run:

`NODE_ENV=production npm install`